gsa/src/web/graphql/__tests__/wizards.js

```diff
@@ -161,3 +154,199 @@ describe('useRunQuickFirstScan tests', () => {
     );
   });
 });
+
+const RunModifyTaskComponent = ({alertEmail, reschedule}) => {
+  const [runModifyTask] = useRunModifyTask();
+  const [ok, setOk] = useState(false);
+  const [error, setError] = useState();
+
+  const handleRunModifyTask = () => {
+    runModifyTask({
+      alertEmail,
+      startDate,
+      startTimezone,
+      reschedule,
+      tasks: [
+        {
+          name: 'myFirstTask',
+          id: '13579',
+          alerts: [{id: '34567'}],
+        },
+      ],
+    })
+      .then(() => setOk(true))
+      .catch(err => setError(err.message));
```

Comment:
you don't need to pass err.message here. The Error class implements to toString() method that returns the message. Therefore the following code would also fit
```suggestion
      .catch(setError);
```